src/components/LearningResourcesWidget/LearningResourcesWidget.tsx

```diff
@@ -54,7 +54,7 @@ const LearningResourcesWidget: React.FunctionComponent<{
   };
 
   return (
-    <div>
+    <div className="learning-resources">
```

Comment:
Any chance we can namespace this as `learning-resources-widget`instead?

Reply:
Done. The whole thing, from the widget body down, is scoped under `learningResources-learningResources`, incidentally .